docs/search.rst

```diff
@@ -0,0 +1,170 @@
+:Authors:
+    Mike Cantelon
+
+Search API
+==========
+
+In addition to the search functionality present in the web interface, the
+storage service also includes a REST search API. Searches are performed by
+sending an HTTP GET request.
+
+Search results will include a count of how many items were found and will
+include next and previous properties indicating links to more items in the
+result set.
+
+Location search
+---------------
+
+The endpoint for searching locations is:
+
+::
+
+  http://<storage service URL>/api/v2/search/location/
+
+Locations can be searched using the following search parameters:
+
+* uuid (location UUID)
+* space (space UUID)
+* purpose (purpose code)
+* enabled (whether the location is enabled)
+
+Adding "?format=json" when sending a GET request will return JSON results.
```

Comment:
While this is true in TastyPie, it's more of a bug than a feature. Not requiring `?format=json` would be better, since we're making improvements.